portal-api/src/cron-scheduler-jobs/helpers/utilisation-report-helpers.js

```diff
@@ -111,20 +112,25 @@ const getEmailRecipient = (paymentOfficerTeam, bankName) => {
  * @returns {Promise<void>}
  */
 const sendEmailForBank = async ({ emailDescription, sendEmailCallback, bank }) => {
-  const { name: bankName, paymentOfficerTeam } = bank;
-  const paymentOfficerTeamEmail = paymentOfficerTeam?.email;
+  const { name: bankName, paymentOfficerTeam, id: bankId } = bank;
+  const paymentOfficerTeamEmails = paymentOfficerTeam?.emails;
 
   try {
-    if (!hasValue(paymentOfficerTeamEmail)) {
-      console.warn(`Not sending ${emailDescription} email to '${bankName}' - no payment officer team email set`);
-    } else if (!isValidEmail(paymentOfficerTeamEmail)) {
-      console.error(`Failed to send ${emailDescription} email to '${bankName}' - invalid payment officer team email '${paymentOfficerTeamEmail}'`);
+    if (!isArray(paymentOfficerTeamEmails) || !paymentOfficerTeamEmails.length) {
+      console.warn(`Not sending ${emailDescription} email to '${bankName}' - no payment officer team emails set`);
     } else {
-      await sendEmailCallback({
-        emailAddress: paymentOfficerTeamEmail,
-        recipient: getEmailRecipient(paymentOfficerTeam, bankName),
+      paymentOfficerTeamEmails.forEach(async (paymentOfficerTeamEmail) => {
```

Comment:
`async` inside a `forEach` doesn't run in the way you would expect - I'm pretty sure it will put any awaited calls on the microtask queue and then carry on with the rest of the synchronous code before executing. This means that, if any error is thrown by `sendEmailCallback`, it won't be caught in our `try-catch`. Wrapping in a `Promise.all` and using `map` is my preferred way of getting around this :)